charts/stable/openvscode-server/values.yaml

```diff
@@ -3,6 +3,9 @@ image:
   pullPolicy: IfNotPresent
   tag: v1.63.2
 
+portal:
+  path: "/?tkn={{ .Values.env.CONNECTION_TOKEN }}"
```

Comment:
```suggestion
  path: "/?tkn={{ .Values.secret.CONNECTION_TOKEN }}"
```

Reply:
@stavros-k If you reference something that doesn't exist, I doub't its going to work at all ;-)